assets/js/components/Stepper/Step.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Step component.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { STEP_STATUS } from './constants';
+
+export default function Step( { children, title, stepStatus } ) {
+	return (
+		<div className="googlesitekit-step">
+			<h2
+				className={ `googlesitekit-step__title googlesitekit-step__title--${ stepStatus }` }
+			>
+				{ title }
+			</h2>
+			{ stepStatus === STEP_STATUS.ACTIVE && (
+				<div className="googlesitekit-step__content">{ children }</div>
```

Comment:
Looking at the other classes here, I think these should use `googlesitekit-stepper__step` as the prefix rather than `googlesitekit-step`, or are these intentionally different?

Reply:
This was a deliberate choice, with with `Step` being the component I considered it to be the block-level construct. However, on re-reading the BEM docs, with `Step` not intended for use outside of `Stepper` it looks like we should indeed use `googlesitekit-stepper__step` as the prefix here. I'll update these class names. Thanks for pointing this out!